### DISCUSSION TITLE
VERB recognised as AUX in simple sentence with en_core_web_* models

### DISCUSSION THREAD

**Discussion by DavidGerva** [explosion/spaCy#9302] [Help: Model Advice] [answered]
I came across this strange behavior of the spacy models.
This simple sentence:
_"Insect-mediated transmission of viruses is a threat"_

gives this output in displacy both with the small and the large models:
![image](https://user-images.githubusercontent.com/1856249/134934090-97f445f3-b3c3-45b3-bb6a-1b49c0f8ddc1.png)

I know models are not perfect. But this is a behavior I often find with "to be" verbs. So I ask:
- is this something that makes sense to report to spacy or not?
- is this an expected output I have to handle in some way?

thank you

**Comment by polm** [comment, marked as answer]
This is an unfortunate but known limitation of the English models due to the underlying training data not really making an AUX/VERB distinction and the Universal Dependencies definition of the distinction not always being clear anyway. You can read more about it [here](https://github.com/explosion/spaCy/discussions/8856#discussioncomment-1150087).
